Integrations/integration-ProofpointTAP.yml

```diff
@@ -0,0 +1,280 @@
+description: Proofpoint's Targeted Attack Protection (TAP) helps protect against and
+  provide additional visibility into phishing and other malicious email attacks.
+configuration:
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: 'A string specifying which threat type will be returned in the data. If
+    no value is specified, all threat types are returned. The following values are
+    accepted: url, attachment, messageText'
+  name: threatType
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: 'A string specifying which threat statuses will be returned in the data.
+    If no value is specified, active and cleared threats are returned. The following
+    values are accepted: active, cleared, falsePositive'
+  name: threatStatus
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Server URL (TAP API URL)
```

Comment:
The parameters order is a bit odd. In my opinion, `Server URL` and `API Version` should be above `Credentials`. Also, there should be example for `Server URL` in its description.